sites/public/pages/resources.tsx

```diff
@@ -0,0 +1,41 @@
+import { PageHeader, t } from "@bloom-housing/ui-components"
+import Layout from "../layouts/application"
+
+const Feedback = () => {
+  const pageTitle = <>{t("pageTitle.resources")}</>
+
+  return (
+    <Layout>
+      <PageHeader title={pageTitle} inverse />
+      <div className="px-5 max-w-5xl md:m-auto">
+        <p className="mt-8">{t("resources.body1")}</p>
+        <div className="mt-8">
```

Comment:
Is there value adding these sections with no accompanying text or links? Maybe we could add them later when they contain more information?

Reply:
This is what Safiya requested for now so I think it's useful to:

1. Get at least the headings in for translations
2. Act as an outline/reminder for what information we still need to gather for this page